README.md

```diff
@@ -1,2 +1,236 @@
-# open-ce-environments
-Collection of Open-CE environment files
+
```

Comment:
I feel like most of this should actually be stored with the build tools. Someone should be able to use the build tools without looking at this repo.

Reply:
Yeah, that's a good point.
This readme  could contain a link to that document.

Maybe this readme should focus on tracking versions of packages?

Reply:
Yes, I think that might be a good idea, although I'm not sure how hard that will be to maintain.

Reply:
I think this is something we could do automatically.
I'll look into this in conjunction with open-ce/open-ce#241